quinn-proto/src/config/mod.rs

```diff
@@ -585,8 +585,8 @@ impl ClientConfig {
 
     /// Set a custom [`TokenStore`]
     ///
-    /// Defaults to [`NoneTokenStore`], which disables the use of tokens from NEW_TOKEN frames as a
-    /// client.
+    /// Defaults to a default [`TokenMemoryCache`], which is suitable for most internet
```

Comment:
Nit: "Defaults to a default" seems a little ugly? I'd probably omit "a default", since I think that is the reasonable assumption anyway.